### DISCUSSION TITLE
Conditional UI for autocomplete suggestions of existing passkeys

### DISCUSSION THREAD

**Discussion by aravindw3f** [MasterKale/SimpleWebAuthn#547] [General]
Is anyone working on integration of the Conditional UI for autocomplete suggestions of existing passkeys?

I wanted to confirm before i get started on a possible solution for it unless it has already been taken up by the core team; or anyone else, with whom i could collaborate with if possible.

I hoping to start making contribution in open source projects and give back to the community.
Any general advice in this aspect would also be very valuable.

I believe it would be great to be able to contribute to functionality and this library.

My apologises if this is not the right forum.

**Comment by MasterKale** [comment]
Hello @aravindw3f, thank you for your interest in contributing. This project is not currently open for external contributions, however if @simplewebauthn/browser's `browserSupportsWebAuthnAutofill()` method isn't working properly for some reason then please create a new issue documenting the problem so that I can fix it:

https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/browser/src/helpers/browserSupportsWebAuthnAutofill.ts
